### ISSUE TITLE
Missing dependency Role::Tiny in version 0.007

### ISSUE THREAD

**Issue by perlpunk** [tobyink/p5-role-hooks#3]
https://build.opensuse.org/package/live_build_log/devel:languages:perl:CPAN-R/perl-Role-Hooks/openSUSE_Tumbleweed/x86_64
```
[    8s] Can't locate Role/Tiny.pm in @INC (you may need to install the Role::Tiny module) (@INC contains: /home/abuild/rpmbuild/BUILD/Role-Hooks-0.007/blib/lib /home/abuild/rpmbuild/BUILD/Role-Hooks-0.007/blib/arch /usr/lib/perl5/site_perl/5.36.0/x86_64-linux-thread-multi /usr/lib/perl5/site_perl/5.36.0 /usr/lib/perl5/vendor_perl/5.36.0/x86_64-linux-thread-multi /usr/lib/perl5/vendor_perl/5.36.0 /usr/lib/perl5/5.36.0/x86_64-linux-thread-multi /usr/lib/perl5/5.36.0 /usr/lib/perl5/site_perl .) at t/99nonrole.t line 4.
[    8s] BEGIN failed--compilation aborted at t/99nonrole.t line 4.
```

**Comment by tobyink** [comment]
Thanks. Fixed and released 0.008.

**Comment by perlpunk** [comment]
thanks!
